### PR TITLE
Update amazon-sqs-connector-release-notes-mule-4.adoc

### PR DESCRIPTION
**Pull request by patrik-patinak**

### CODE REVIEW

File: modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc [hunk 1]
```diff
@@ -9,6 +9,23 @@ The Amazon SQS cloud connector provides an easy way to interface with the Amazon
...
+
+[%header%autowidth.spread]
+|===
+|Application/Service|Version
+|Mule Runtime|4.1.0 and later
+|AWS SDK for Java|1.11.515
+|===
+
+=== Fixed in this Release
+* Receive message source was not using the default global queue URL when queue URL was not specified, consequently an error was thrown after the application deploy. This version fixes this issue. [SE-12178]
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc]: Receive message source did not use the default global queue URL when the queue URL was not specified. Therefore, an error was thrown after the application deployed.  [SE-12178]

File: modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc [hunk 2]
```diff
@@ -9,6 +9,23 @@ The Amazon SQS cloud connector provides an easy way to interface with the Amazon
```
Comment (dhirsch1) [inline modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc]: Please add "Mule 4" to the title of the release notes: "Amazon SQS Connector Release Notes - Mule 4"

Comment (patrik-patinak) [reply modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc]: Added the Mule version

File: modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc (hunk 1 above)
Comment (patrik-patinak) [reply modules/ROOT/pages/connector/amazon-sqs-connector-release-notes-mule-4.adoc]: Changed the fix description

**Pull request merged**